main/ui/src/test/java/org/cryptomator/ui/recoverykey/RecoveryKeyFactoryTest.java

```diff
@@ -48,17 +48,17 @@ public void testValidateValidateRecoveryKeyWithTooShortInput() {
 	@Test
 	@DisplayName("validateRecoveryKey() with invalid checksum")
 	public void testValidateValidateRecoveryKeyWithInvalidCrc() {
-		boolean result = inTest.validateRecoveryKey("pathway lift abuse plenty export texture gentleman landscape beyond ceiling around leaf cafe" +
-				" charity border breakdown victory surely computer cat linger restrict infer crowd live computer true written amazed investor boot" +
+		boolean result = inTest.validateRecoveryKey("pathway lift abuse plenty export texture gentleman landscape beyond ceiling around leaf cafe" + //
+				" charity border breakdown victory surely computer cat linger restrict infer crowd live computer true written amazed investor boot" + //
```

Comment:
No that important because we will get String literals soon, but can we have the `+` at the begin of a line?

Kind of like virtually nobody would write `foo(). \n bar()`.

Reply:
Resolved by a5bac33a279b9756cc892d7754620366c5219250.